adc/microphone_adc/README.adoc

```diff
@@ -4,12 +4,12 @@ This example code shows how to interface the Raspberry Pi Pico with a standard a
 
 [TIP]
 ======
-An analog to digital converter (ADC) is responsible for reading continuous input signals that may range from 0 to a specified reference voltage (in the Pico's case this reference voltage is set by the supply voltage and can be measured on pin 35, ADC_VREF) and converting them into 1s and 0s, ie. a number that can be digitally stored.
+An analog to digital converter (ADC) is responsible for reading continuous input signals that may range from 0 to a specified reference voltage (in the Pico's case this reference voltage is set by the supply voltage and can be measured on pin 35, ADC_VREF) and converting them into binary, ie. a number that can be digitally stored.
```

Comment:
I wonder if "continually varying" might be a good substitute for "continuous" here?

@aallan should the "ie." be "i.e." ?

Reply:
> @aallan should the "ie." be "i.e." ?

Yes.